### PR TITLE
p0-signin-signup-home-product-pages

### PR DESCRIPTION
**Pull request by PuneetP16**
## Login Page
image

## SIgnup Page
image

## Products Page
image

image

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,11 +23,471 @@
 		<!-- Magnificent-UI library -->
```
Comment (tarunsankhla) [inline index.html:23]: in the preview category item list the spacing/gap is not even

File: index.html [hunk 2]
```diff
@@ -23,11 +23,471 @@
...
+				/>
+
+				<button type="submit" class="btn btn--primary btn--icon">
+					<i class="bx bx-search-alt-2"></i>
+				</button>
+			</form>
+			<div class="header__nav_btns">
+				<a class="btn btn--primary" href="/pages/signin/signin.html">Login</a>
+				<a
+					class="header__links badge_base btn btn--primary btn--icon btn--circular"
```
Comment (tarunsankhla) [inline index.html:52]: a common practice is to follow only one form of class name either snake or dash

File: pages/products/products.css [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+}
+
+.nav__items {
+	list-style: none;
+	width: 100%;
+	gap: 0.6rem;
+}
+
+.nav__list {
+	width: 120px;
```
Comment (tarunsankhla) [inline pages/products/products.css]: try using rem bcoz every where you have used them

File: pages/signin/signin.css [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+	display: flex;
+	place-content: center;
+}
+
+.input_box {
+	font-size: 1.6rem;
+	width: 100%;
+	border-radius: 4px;
+	padding: 0.5rem 1rem;
+	border: 1px solid #c5c5c5;
```
Comment (tarunsankhla) [inline pages/signin/signin.css:52]: create variable of this hex

File: styles/base-style.css [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+@media only screen and (max-width: 400px) {
+	:root {
+		font-size: 60%;
+	}
+
+	.nav__links {
+		width: 10rem;
+	}
+}
```
Comment (tarunsankhla) [inline styles/base-style.css:59]: a good practise is to have media query atlast

File: pages/homepage/homepage.html [hunk 6]
```diff
@@ -0,0 +1,491 @@
...
+			href="https://magnificentui.netlify.app/src/magnificent.css"
+		/>
+
+		<!-- Store-CSS -->
+		<link rel="stylesheet" href="/styles/style.css" />
+	</head>
+	<body class="body--homepage">
+		<header class="header">
+			<a class="brand__text grid-center" href="/pages/homepage/homepage.html"
+				>Magnificent Store</a
```
Comment (pankaj-prog) [inline pages/homepage/homepage.html]: there's no need of grid center, cause you already have header as flex and you have given align-items center to it.

File: pages/homepage/homepage.css [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+
+/* categories section */
+.homepage__section {
+	display: flex;
+	flex-flow: column nowrap;
+	gap: 3rem;
+	align-items: center;
+}
+
+.categories__items {
```
Comment (rahulyadav139) [inline pages/homepage/homepage.css:23]: Everything looks great to me. Go ahead and merge the branch.

**Review by rahulyadav139 — APPROVED** [review]

File: pages/homepage/homepage.html (hunk 6 above)
Comment (Punit2116) [reply pages/homepage/homepage.html]: Thanks for mentioning, I kept that in div before so forgot to remove grid center when removed from div.

**Comment by sushmita12345** [comment]
Suggest to use 'px' if size is below 1rem and in .nav__list you can use 'rem' instead of 'px'.
Looks good to merge.

Screenshot 2022-02-14 at 11 39 54 AM

**Review by sushmita12345 — APPROVED** [review]

File: pages/signin/signin.css [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+
+.center {
+	height: 100%;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+}
+
+a {
+	text-decoration: none;
```
Comment (m26unkwn) [inline pages/signin/signin.css:11]: you can use all:unset property here

**Review by m26unkwn — APPROVED** [review]
Otherwise everything is good to merge.

File: pages/signin/signin.css (hunk 8 above)
Comment (PuneetP16) [reply pages/signin/signin.css:11]: I will look into it.

**Review by dhruvin25799 — APPROVED** [review]

File: pages/products/products.css (hunk 3 above)
Comment (tarunsankhla) [reply pages/products/products.css]: Knock knock

File: index.html (hunk 2 above)
Comment (PuneetP16) [reply index.html:52]: I am following BEM:
block_name__element_name--modifier_name

File: pages/products/products.css (hunk 3 above)
Comment (PuneetP16) [reply pages/products/products.css]: Yup surely will update

File: pages/signin/signin.css (hunk 4 above)
Comment (PuneetP16) [reply pages/signin/signin.css:52]: Oh thanks it got missed out.

File: styles/base-style.css (hunk 5 above)
Comment (PuneetP16) [reply styles/base-style.css:59]: Noted.

**Pull request merged**